app/routers/exercise.py

```diff
@@ -0,0 +1,48 @@
+from fastapi import APIRouter, Depends, Request
+from app.database.models import User
+from app.dependencies import get_db, templates
+from app.routers.user_exercise import get_user_exercise
+from datetime import datetime
+from app import config
+from sqlalchemy.orm import Session
+
+router = APIRouter(
+    prefix="/exercise",
+    tags=["exercise"],
+    responses={404: {"description": "Not found"}},
+)
+
+
+@router.get("/")
+async def exercise(
+        request: Request,
+        session: Session = Depends(get_db),
+        ):
+    """
+    If is active exercise = True
+    Show user exercise for a specific day if is_active_exercise is ture.
+    """
+    user = session.query(User).filter_by(id=1).first()
+    if not user:
+        # create empty default user
+        user = User(
+            username='',
+            password='',
+            email=''
+        )
+    # Get user exercise
+    user_exercise = get_user_exercise(session, user_id=user.id)
+    if user_exercise:
+        # Get exercise day
+        date_time_now = datetime.now()
+        delta = date_time_now - user_exercise[0].start_date
+        # All exercises split to 30 days
+        day = (delta.days % 30) + 1
```

Comment:
Why `% 30`?

Reply:
30 days exercise 

I have 30 playing days so in the day 31 I will return to day 1

Reply:
So please save it as a constant and give it an indicative name.